extractor/server/server.go

```diff
@@ -39,6 +46,26 @@ func main() {
 	}
 }
 
+func parseMaxMessageSize(maxMessageSize string) int {
```

Comment:
I can not see right now, how that can be equivalent. I.e how would  that allow to set a maximum possible value?